plugin/src/main/java/com/playmonumenta/scriptedquests/zones/zone/ZoneFragment.java

```diff
@@ -15,17 +17,28 @@
  */
 public class ZoneFragment<T> extends BaseZone {
 	private HashMap<String, Zone<T>> mParents = new HashMap<String, Zone<T>>();
+	private HashMap<String, ArrayList<Zone<T>>> mParentsAndEclipsed = new HashMap<String, ArrayList<Zone<T>>>();
```

Comment:
A lot of these ArrayList declarations should really just be List. I think you changed this in a lot of spaces for LinkedHashSet -> Set, etc. 

Not important, can skip this to merge this. Just a note for future.